mpp/build.gradle.kts

```diff
@@ -214,6 +220,48 @@ tasks.register("testUIKit") {
     dependsOn(":collection:collection:$uikitTestSubtaskName")
 }
 
+/**
+ * Prints all darwin(ios/apple) related APIs from "*.klib.api" files.
+ * Use this task after `./gradlew klibApiDump`
+ */
+tasks.register("printDarwinOnlyKlibApi") {
```

Comment:
Exactly, we have to go through the review first, then we can use it for validation.

Reply:
> Common code validates with desktop dump, so we can think if we need this redundancy.

It is validated via Java Bytecode, not via KLib. It can be accidentaly broken

Reply:
This command is temporary useful. Maybe just keep in a separate branch?